packages/gatsby-telemetry/src/telemetry.js

```diff
@@ -19,11 +19,37 @@ module.exports = class AnalyticsTracker {
   constructor() {
     try {
       this.componentVersion = require(`../package.json`).version
+      this.installedGatsbyVersion = this.getGatsbyVersion()
+      this.gatsbyCliVersion = this.getGatsbyCliVersion()
     } catch (e) {
       // ignore
     }
   }
 
+  getGatsbyVersion() {
+    const packageInfo = require(join(
+      process.cwd(),
+      `node_modules`,
+      `gatsby`,
+      `package.json`
+    ))
+    try {
+      return packageInfo.version
+    } catch (e) {
+      // ignore
+    }
+    return undefined
+  }
+
+  getGatsbyCliVersion() {
+    try {
+      const { version } = require(`../package.json`)
```

Comment:
@jamo this seems a copy of our current code but it doesn't reflect cli version but telemetry version. You probably want to do some kind of voodoo here to get the correct path of gatsbycli.

this is just some pseudo code as I run `yarn build`
```
process.argv[1] = "/home/<user>/<somepath>/starters/node_modules/.bin/gatsby"
let realBinPath = process.argv[1];
if (isSymlink(realBinPath)) {
  realBinPath = fs.resolveSymlink(realBinPath) // unsure if this exists :D
}

const { version }traverseDownForPackageJSON(realBinPath)
```

Reply:
indeed 🤦‍♂️